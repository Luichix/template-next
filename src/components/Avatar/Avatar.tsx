import { AvatarProps } from './AvatarTypes'

export const Avatar = ({ uri }: AvatarProps) => {
  return (
    <img
      src={uri}
      style={{
        height: '68px',
        width: '68px',
        borderRadius: '35px',
        marginRight: '10px',
      }}
    />
  )
}

export default Avatar
