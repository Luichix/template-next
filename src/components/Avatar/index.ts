export { default, Avatar } from './Avatar'
