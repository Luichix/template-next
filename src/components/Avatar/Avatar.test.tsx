import renderer from 'react-test-renderer'
import Avatar from './Avatar'

it('renders snapshot atoms avatar', () => {
  const tree = renderer
    .create(
      <Avatar uri="https://notjustdev-dummy.s3.us-east-2.amazonaws.com/avatars/10.png" />
    )
    .toJSON()
  expect(tree).toMatchSnapshot()
})
