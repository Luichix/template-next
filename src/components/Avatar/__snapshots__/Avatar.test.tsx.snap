// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders snapshot atoms avatar 1`] = `
<img
  src="https://notjustdev-dummy.s3.us-east-2.amazonaws.com/avatars/10.png"
  style={
    {
      "borderRadius": "35px",
      "height": "68px",
      "marginRight": "10px",
      "width": "68px",
    }
  }
/>
`;
